modules/save-generated-policies.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-network-policies.adoc
+:_module-type: PROCEDURE
+[id="save-generated-policies_{context}"]
+= Saving generated policies
+
+[role="_abstract"]
+You can download and save the generated network policies from {product-title}.
+Use this option to commit the policies into a version control system like Git.
+
+.Procedure
+* Select the *Download YAML* icon on the *Network Policy Simulator* panel.
++
+[NOTE]
+====
+. To create policies using the saved YAML file, use the following command:
```

Comment:
I know I haven't been pointing out notes/admonitions with bullets and procedures, but this one is kind of unnecessarily long for the information it's actually conveying. Suggest condensing it.